### PR TITLE
Use html instead of hooks to figure out if a shortcut should be executed

### PR DESCRIPTION
**Pull request by vonElfvin**
# Jira Issue: [IP-37]

## What?
- Use html instead of hooks+event listeners to figure out if a shortcut should be executed
- Add KeyCode map (that I didn't end up using 🤦) that I figure we might want some day 🤷 

## Why?
- This should be more reliable when it comes to modifiers

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [x] Codescouted
- [ ] Unit tests written
- [x] Tested locally

[IP-37]: https://hedvig.atlassian.net/browse/IP-37

### CODE REVIEW

File: src/utils/hooks/command-line-hook.tsx [hunk 1]
```diff
@@ -326,15 +301,60 @@ export const CommandLineProvider: React.FC = ({ children }) => {
...
+  hide: () => void
+}> = ({ actions, hide, children }) => {
+  const [actionKeyCodes, setActionKeyCodes] = useState<number[][]>([])
+  useEffect(() => {
+    setActionKeyCodes(
+      actions.map((action) => action.keys.map((key) => key.code)),
+    )
+  }, [actions])
+
+  // tslint:disable:no-unused-expression
```
Comment (rasmusguterstam) [inline src/utils/hooks/command-line-hook.tsx:333]: what why

Comment (vonElfvin) [reply src/utils/hooks/command-line-hook.tsx:333]: `e.shiftKey && modifiers.push(Keys.Shift.code)` got complaints from tslint

File: src/utils/hooks/command-line-hook.tsx [hunk 2]
```diff
@@ -326,15 +301,60 @@ export const CommandLineProvider: React.FC = ({ children }) => {
...
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLDivElement>) => {
+    const modifiers: number[] = []
+    e.shiftKey && modifiers.push(Keys.Shift.code)
+    e.ctrlKey && modifiers.push(Keys.Control.code)
+    e.altKey && modifiers.push(Keys.Option.code)
+    e.metaKey && modifiers.push(Keys.Command.code)
```
Comment (rasmusguterstam) [inline src/utils/hooks/command-line-hook.tsx:339]: Nice, very clean!

**Review by rasmusguterstam — APPROVED** [review]
Looks good, and the few tests we have seem to work as well! 😄

**Pull request merged**